language/libra-vm/src/errors.rs

```diff
@@ -78,19 +96,30 @@ pub fn convert_normal_prologue_error(status: VMStatus) -> VMStatus {
 /// `UNEXPECTED_ERROR_FROM_KNOWN_MOVE_FUNCTION`
 pub fn convert_normal_success_epilogue_error(status: VMStatus) -> VMStatus {
     match status {
-        VMStatus::MoveAbort(location, code @ EACCOUNT_FROZEN)
-        | VMStatus::MoveAbort(location, code @ EINSUFFICIENT_BALANCE) => {
-            if location != known_locations::account_module_abort() {
+        VMStatus::Executed => VMStatus::Executed,
+        VMStatus::MoveAbort(location, code)
+            if location != known_locations::account_module_abort() =>
+        {
+            let (category, reason) = error_split(code);
+            error!(
+                "[libra_vm] Unexpected success epilogue Move abort: {:?}::{:?} \
```

Comment:
...and also drop the "success" in this message

Reply:
see above